superset/errors.py

```diff
@@ -15,7 +15,7 @@
 # specific language governing permissions and limitations
 # under the License.
 # pylint: disable=too-few-public-methods,invalid-name
-from dataclasses import dataclass
+from dataclasses import dataclass  # pylint: disable=wrong-import-order
```

Comment:
don't know why but every linter decided to lose its shit here.

Reply:
What version of Python are you on? I'm guessing the ordering fails due to dataclasses being built-in on 3.7 but an addon in previous versions. But no need to fix here, I can clean this up later.

Reply:
I'm on 3.7